src/test/ui/span/issue-43927-non-ADT-derive.stderr

```diff
@@ -0,0 +1,8 @@
+error: `derive` may only be applied to structs, enums, and unions
+  --> $DIR/issue-43927-non-ADT-derive.rs:13:1
+   |
+13 | #![derive(Debug, PartialEq, Eq)] // should be an outer attribute!
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This is a great example of a case where a `note` pointing out that this derive is being applied to the outer context instead of the `struct` immediately after would be beneficial. If you don't want to add that on this PR, could you please file a ticket for it?

It could be something along the lines of

```
error: `derive` may only be applied to structs, enums, and unions
  --> $DIR/issue-43927-non-ADT-derive.rs:13:1
   |
13 | #![derive(Debug, PartialEq, Eq)]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ derive cannot be applied to outer mod
note: as it is written, the derive is applied to the outer scope, did you mean to apply it to the next element?
   |
13 | #[derive(Debug, PartialEq, Eq)]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

```

Reply:
083f053